docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version:
+services:
```

Comment:
Sorry, but what functionality will create a database structure?

Reply:
Please look at here again. About docker helped to me from our group Marina.